src/devices/acurite.c

```diff
@@ -852,3 +943,14 @@ r_device acurite_606 = {
     .disabled       = 0,
     .demod_arg      = 0,
 };
+
+r_device acurite_00275rm = {
+    .name           = "Acurite 00275rm,00276rm Temp/Humidity with optional probe",
+    .modulation     = OOK_PULSE_PWM_TERNARY,
+    .short_limit    = 320,  // = 4* 80,  80  is reported by -G option
+    .long_limit     = 520,  // = 4*130, 130  "
+    .reset_limit    = 608,  // = 4*152, 152  "
+    .json_callback  = &acurite_00275rm_callback,
+    .disabled       = 1,
```

Comment:
Since this outputs structured data and doesn't look experimental, I believe this should be enabled by default (disabled = 0).